src/ActiveLogin.Authentication.BankId.Api/BankIdApiClientExtensions.cs

```diff
@@ -48,6 +48,58 @@ public static Task<AuthResponse> AuthAsync(this IBankIdApiClient apiClient, stri
             return apiClient.AuthAsync(new AuthRequest(endUserIp, personalIdentityNumber));
         }
 
+        /// <summary>
```

Comment:
See comment below on overloads for construvtor on SignRequest. The same I'd say apply here, but maybe we don't need all overloads?

Suggestion:
- `public SignRequest(string endUserIp, string userVisibleData)`
- `public SignRequest(string endUserIp, string userVisibleData, string userNonVisibleData)`
- `public SignRequest(string endUserIp, string userVisibleData, string userNonVisibleData, string personalIdentityNumber)`